system-int-test/test/cmr/system_int_test/ingest/collection_misc_validation_test.clj

```diff
@@ -77,6 +78,53 @@
       (is (some? (re-find #"/Purpose string.*is too long \(length: 12000, maximum allowed: 10000\)" (:warnings ingest-response))))
       (is (some? (re-find #"/Purpose string.*is too long \(length: 12000, maximum allowed: 10000\)" (:warnings validation-response)))))))
 
+(deftest error-messages-are-friendly
+  (testing "Error messages don't contain regexes"
+    ;; Include fields that will fail the regex pattern and ensure that the regexes
+    ;; don't show up in the returned errors
+    (let [platform-collection (data-umm-c/collection
+                               {:Platforms [(data-umm-cmn/platform {:ShortName ""
+                                                                    :LongName ""})]})
+          data-center-collection (data-umm-c/collection
+                                  {:DataCenters [(data-umm-cmn/data-center {:ShortName "asdf"
+                                                                            :Roles ["ARCHIVER"]})]})
+          science-keyword-collection (data-umm-c/collection
+                                      {:ScienceKeywords
+                                       [(data-umm-cmn/science-keyword
+                                         {:VariableLevel1 ""
+                                          :VariableLevel2 ""
+                                          :VariableLevel3 ""
+                                          :Category ""
+                                          :Term ""
+                                          :DetailedVariable ""
+                                          :Topic ""})]})]
+      (are3 [collection]
+       (let [ingest-response (d/ingest-umm-spec-collection "PROV1" collection {:format :umm-json :allow-failure? true})
+             validation-response (ingest/validate-concept (data-umm-c/collection-concept collection :umm-json))]
+         (is (nil? (first (map #(re-find #"(ECMA|regex)" %) (:errors ingest-response)))))
+         (is (nil? (first (map #(re-find #"(ECMA|regex)" %) (:errors validation-response)))))
+         (is (some? (map #(re-find #"is an invalid format" %) (:errors ingest-response))))
```

Comment:
These tests don't look like they've been updated to include the word "in". Are they failing?